apps/envited-x-data-space/common/asset/utils.ts

```diff
@@ -129,16 +132,23 @@ export const hasManifestThirdPartyLinks = (manifest: Manifest) =>
 
 export const getPathsFromManifestLinks = (links: ManifestLink[]) =>
   pipe(
-    map((link: ManifestLink) =>
-      !isRemoteUrl(pathOr('', MANIFEST_LINK_FILE_PATH)(link))
+    map((link: ManifestLink) => {
+      const fileType = path(MANIFEST_CATEGORY_ID)(link) as string
+      const fileData = !isRemoteUrl(pathOr('', MANIFEST_LINK_FILE_PATH)(link))
         ? {
             path: formatManifestLinkPath(pathOr('', MANIFEST_LINK_FILE_PATH)(link)),
-            type: path(MANIFEST_CATEGORY_ID)(link),
+            type: fileType,
           }
-        : null,
-    ),
+        : null
+      if (fileData && isMedia(fileType)) {
+        const mimeType = pathOr('', MANIFEST_LINK_MIME_TYPE)(link)
+        return assoc('mimeType', mimeType)(fileData)
```

Comment:
Do we want only a mimeType when it's media, do we not want to have this for all the files that are in a asset?